docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-immutable-con.adoc

```diff
@@ -0,0 +1,15 @@
+[id='environment-immutable-con']
+= Environment with immutable servers
+You can deploy an environment that includes several different pods running {KIE_SERVER} with preloaded processes. The database servers are, by default, also run in pods. Each {KIE_SERVER} pod can be separately scaled as necessary.
+
+In this case, any processes (KJAR files) must be loaded onto a {KIE_SERVER} at the time the image is created. You can not load or unload processes on a running {KIE_SERVER}. The advantage of this approach is that the {KIE_SERVER} with the processes in it runs like any other containerized service and does not require specialized management. You can use existing approaches to cloud integration to deploy and manage the servers.
+
+Optionally, you can also deploy a pod with {CENTRAL} Monitoring and pod with Smart Router are also deployed. You can optionally use {CENTRAL} Monitoring to start and stop (but not deploy) processes on your {KIE_SERVERS} and to view monitoring data. 
+
+Smart Router is a single endpoint that can receive calls to any of your processes and route them automatically to the server that actually runs the the process.
+
+image::openshift-pam-managed.png[Architecture diagram of an infrastructure with immutable pods, Smart Router, and {CENTRAL} Monitoring]
+
+When you create a {KIE_SERVER} image, you must build your processes using Source to Image (S2I). Provide a Git repository with the source of your decision service; you can use the Git repository built into your authoring {CENTRAL}. OpenShift automatically builds the source, installs the decision service into the {KIE_SERVER} image, and starts the service. No further management of the image is required. If you want to use a new version of the decision service, you can build a new image.
+
+If you want to use {CENTRAL} Monitoring, you must install the Monitoring and Smart Router template _before_ creating any {KIE_SERVER} images. You must also provide a Maven repository. Your integration process must ensure that all the versions of KJAR files built into any {KIE_SERVER} image are also available in the Maven repository.
```

Comment:
you must install the Monitoring and Smart Router template -> you must install the Monitoring, and optionally also the Smart Router template,

Reply:
@bmozaffa there is no smart router template at https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/rhpam70-dev/templates so I just assumed the monitoring template also includes the smart router - if that is not so, how does this actually work?

Reply:
You are correct and that is the case. I'm just implying that users may choose to remove smart router from the template and deploy an environment that does not include it at all.